components/editor/toolbar.md

```diff
@@ -33,22 +33,64 @@ To control the collection of buttons and commands available to the user, you pro
 
 ## Built-in Tool Lists
 
-The Editor comes with two predefined sets of tools in the `EditorToolSets` static class:
+The Editor comes with [two predefined sets of tools](#predefined-toolset-configurations) in the `EditorToolSets` static class:
 
-* `Default` - the default set of the most commonly used tools and commands.
-* `All` - All the available tools and commands the editor has.
+* `Default` includes the most commonly used tools and commands. If you do not apply any settings, the `Default` list of tools will be used.
+* `All` includes all the available tools and commands.
 
-If you do not apply any settings, the `Default` list of tools will be used.
+The following example shows how to use the `All` toolset.
 
->caption Switch to the All tools built-in toolset
+>caption Use all built-in Editor tools
 
 ````CSHTML
 @using Telerik.Blazor.Components.Editor
 
 <TelerikEditor Tools="@EditorToolSets.All">
 </TelerikEditor>
+
+@code {
+    private string EditorValue { get; set; } = string.Empty;
```

Comment:
Is this property required for the example?

Reply:
Yes, it's good to not provide invalid runnable configurations. I should add `@bind-Value`. Thanks!